### PR TITLE
Adding Contrail/Tungsten Fabric to the add-on list

### PR DESCRIPTION
**Pull request by aniketgawade**
Added section on OpenContrail/Tungsten Fabric with a brief description.

### CODE REVIEW

**Comment by Bradamant3** [comment]
@aniketgawade Thanks for your PR, but we need you to sign the CLA before we can merge it.

File: content/en/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Add-ons in each section are sorted alphabetically - the ordering does not imply
...
 * [CNI-Genie](https://github.com/Huawei-PaaS/CNI-Genie) enables Kubernetes to seamlessly connect to a choice of CNI plugins, such as Calico, Canal, Flannel, Romana, or Weave.
 * [Contiv](http://contiv.github.io) provides configurable networking (native L3 using BGP, overlay using vxlan, classic L2, and Cisco-SDN/ACI) for various use cases and a rich policy framework. Contiv project is fully [open sourced](http://github.com/contiv). The [installer](http://github.com/contiv/install) provides both kubeadm and non-kubeadm based installation options.
+* [Contrail](http://www.juniper.net/us/en/products-services/sdn/contrail/contrail-networking/), based on [Tungsten Fabric](https://tungsten.io), is a truly open, multi-cloud network virtualization and policy management platform. Contrail and Tungsten Fabric are integrated with various orchestration systems such as Kubernetes, OpenShift, OpenStack and Mesos, and provide different isolation modes for virtual machines, containers/pods and bare metal workloads.
```
Comment (Bradamant3) [inline content/en/docs/concepts/cluster-administration/addons.md]: (in addition to signing the CLA) recommend removing adverbs and adjectives that don't add value to techdoc:
"truly", "various", "different"
Thanks!

**Comment by cody-clark** [comment]
Hi @aniketgawade! Have you had a chance to sign the CLA yet?

>  Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.

File: content/en/docs/concepts/cluster-administration/addons.md (hunk 1 above)
Comment (aniketgawade) [reply content/en/docs/concepts/cluster-administration/addons.md]: Fixed it!

Comment (aniketgawade) [reply content/en/docs/concepts/cluster-administration/addons.md]: Also signing CLA is done. Thank you.

**Pull request merged**